docs/art/retransmit_stage.bob

```diff
@@ -0,0 +1,45 @@
+                                              +------------+
+                                              |  Gossip    |
+                                              |  Service   |
+                                              |            |
+                                              +------------+
+                                                     |Repair peers
+                                                     |
+                          +------------------------------------------------------------------------------+
+                          |                          |                                                   |
+                          |  Retransmit Stage        |                                                   |
+                          |                          |                                                   |
+                          |  +---------------------------------------------+                             |
+                          |  |Window Service         |                     |                             |          +---------+
+                          |  |                       |                     |          +---------------+  |          |         |
+           Packets        |  +---------+  +--------+ |                     |  Shreds  |               |  | Entries  | Replay  |
+          +----------------->+Deshred  +->+Filter  +--------------------------------->+ Blockstore    +------------>+ Stage   |
+                          |  +---------+  ++-+-----+ |                     |          |               |  |          |         |
++--------+                |  |             ^ |       |                     |          |               |  |          +---------+
+|        | Leader Schedule|  |             | |       v                     |          |               |  |
+| Bank   +---------------------------------- |     +-+---------+           |          ++--------------+  |
+|        |                |  |               |     | Repair    |           |           |                 |
++--------+    +----------------------------------->+ Service   | Incomplete|Slots      |                 |
```

Comment:
By Incomplete Slots, do you mean Incomplete Blocks?  A slot is a duration of time.  A block is the thing full of transactions that's sent during a slot.

Reply:
It's telling it the slot index of the slots it needs to ask for repair because they have missing shreds.